packages/server/src/queries/complex/pools/providers/sidecar.ts

```diff
@@ -20,20 +20,24 @@ const poolsCache = new LRUCache<string, CacheEntry>(DEFAULT_LRU_OPTIONS);
 /** Lightly cached pools from sidecar service. */
 export function getPoolsFromSidecar({
   assetLists,
-  chainList,
   poolIds,
+  minLiquidityUsd,
 }: {
   assetLists: AssetList[];
   chainList: Chain[];
   poolIds?: string[];
+  minLiquidityUsd?: number;
 }): Promise<Pool[]> {
   return cachified({
     cache: poolsCache,
-    key: poolIds ? `sidecar-pools-${poolIds.join(",")}` : "sidecar-pools",
+    key:
+      (poolIds ? `sidecar-pools-${poolIds.join(",")}` : "sidecar-pools") +
+      minLiquidityUsd,
     ttl: 5_000, // 5 seconds
     getFreshValue: async () => {
       const sidecarPools = await timeout(
-        () => queryPools({ poolIds }),
+        () =>
+          queryPools({ poolIds, minLiquidityCap: minLiquidityUsd?.toString() }),
```

Comment:
oh nice it's already here

Reply:
Yeah, and it should be back compat so we can release this now then once SQS is released, the var will start being respected on the SQS side